### ISSUE TITLE
Multiple choice selection and settings-import not working in 3.0.2 

### ISSUE THREAD

**Issue by edyso** [Waboodoo/HTTP-Shortcuts#364]
**Steps To Reproduce**

- Start import from File, choose a file (unedited json-file from export file)-> empty Array (import impossible)
- Try to edit a multiple choice variable ends with crash/back to main window
- Call a shortcut with integrated multiple choice variable ends with crash/back to main window

Back to 3.0.1 import is working and no crashes using multiple choice variables.

**Comment by pingu8007** [comment]
```
FATAL EXCEPTION: main
Process: ch.rmy.android.http_shortcuts, PID: 6865
java.lang.IllegalStateException: TypeToken must be created with a type argument: new TypeToken<...>() {}; When using code shrinkers (ProGuard, R8, ...) make sure that generic signatures are preserved.
	at k6.a.<init>(SourceFile:10)
	at ch.rmy.android.http_shortcuts.utils.j.<init>(Unknown Source:0)
	at ch.rmy.android.http_shortcuts.utils.l.a(SourceFile:8)
	at ch.rmy.android.http_shortcuts.data.models.Variable.getDataForType(SourceFile:11)
	at ch.rmy.android.http_shortcuts.activities.variables.editor.types.f1.F(SourceFile:55)
	at ch.rmy.android.http_shortcuts.activities.variables.editor.c1.m(SourceFile:36)
	at ch.rmy.android.framework.viewmodel.d.f(SourceFile:5)
	at ch.rmy.android.http_shortcuts.activities.variables.editor.j0.e(SourceFile:107)
	at ch.rmy.android.framework.data.e.e(SourceFile:61)
	at ch.rmy.android.framework.data.b.e(SourceFile:67)
	at ch.rmy.android.framework.data.i.e(SourceFile:61)
	at kotlinx.coroutines.flow.l.d(SourceFile:146)
	at kotlinx.coroutines.flow.k.i(SourceFile:12)
	at o7.a.f(SourceFile:9)
	at kotlinx.coroutines.j0.run(SourceFile:109)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:223)
	at android.app.ActivityThread.main(ActivityThread.java:7664)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
	Suppressed: kotlinx.coroutines.h0: [v1{Cancelling}@fe11a11, Dispatchers.Main.immediate]
```

Same issue here. Above is the exception thrown when trying to edit a multiple select variable.

**Comment by wallypg** [comment]
Same here, when trying to edit a multiple choice variable it crashes, but also if I try to create a new one when adding the second option.

**Comment by Waboodoo** [comment]
Thanks for the bug report, and sorry for the inconvenience. I will look into it on the weekend and prepare a fix ASAP.

**Comment by Waboodoo** [comment]
These issues should now be fixed in the 3.1.0 release.
